gui-apps/organicmaps/files/fix-3party.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/3party/CMakeLists.txt b/3party/CMakeLists.txt
+index d4996e302d..953787945b 100644
+--- a/3party/CMakeLists.txt
++++ b/3party/CMakeLists.txt
+@@ -41,9 +41,6 @@ if (NOT WITH_SYSTEM_PROVIDED_3PARTY)
+   # Add pugixml library.
+   add_subdirectory(pugixml)
+ 
+-  # Add protobuf library.
+-  add_subdirectory(protobuf)
+-
+   if (NOT PLATFORM_LINUX)
+     add_subdirectory(freetype)
+     add_subdirectory(icu)
+@@ -55,6 +52,9 @@ if (NOT WITH_SYSTEM_PROVIDED_3PARTY)
+   target_include_directories(utf8cpp INTERFACE "${OMIM_ROOT}/3party/utfcpp/source")
+ endif()
+ 
++# Add protobuf library.
++add_subdirectory(protobuf)
++
```

Comment:
Please add a comment here - why do we need this (why not fixed in upstream), and when we can remove this patch.

Reply:
The comment is in the ebuild itself, there is the appropriate PR linked that is needed in organicmaps to make this patch obsolete.